documentation/generated/examples.json

```diff
@@ -12,7 +12,7 @@
       "url": "/guides/blocks/column-formats"
     },
     "exampleFooterLink": "https://coda.io/packs/build/latest/guides/blocks/column-formats",
-    "learnMoreLink": "/guides/blocks/column-formats",
+    "learnMoreLink": "/guides/blocks/column-formats.md",
```

Comment:
I don't think it will work to have the `.md` links in this file, since it's used to power the Help side panel in the Pack Studio.

Instead of changing `documentation_compiler.ts` to add the extension, instead add it in `documentation/example_page_template.md` which only affects the generated markdown here.